.github/workflows/FormatCheck.yml

```
@@ -28,11 +28,8 @@ jobs:
 
       - uses: actions/checkout@v4
       - name: Install JuliaFormatter and format
-        # This will use the latest version by default but you can set the version like so:
-        #
-        # julia  -e 'using Pkg; Pkg.add(PackageSpec(name="JuliaFormatter", version="0.13.0"))'
         run: |
-          julia  -e 'using Pkg; Pkg.add(PackageSpec(name="JuliaFormatter"))'
+          julia  -e 'using Pkg; Pkg.add(PackageSpec(name="JuliaFormatter", version="v1.0.62"))'
```

```suggestion
          julia  -e 'using Pkg; Pkg.add(PackageSpec(name="JuliaFormatter", version="1"))'
```
I think simply restricting to 1 is fine for now, we should definitely rediscuss formatters because the v2 combined with blue style is definitely buggy

---

Aren't we using YAS? I should probably read the v2 patchnotes to see if the changes we are seeing are intentional. Maybe they are desirable. For now I just changed to V1 to not touch the entire repo in this PR.

---

Oh you're right, it depends on the package and we're not necessarily consistent...
Definitely agree with restricting to v1, I just meant that you don't have to restrict the minor versions I think.